queries/pascal/highlights.scm

```diff
@@ -91,7 +78,27 @@
 	(kThen)
 	(kElse)
 ] @conditional
-	
+
+[
+	(kPublished)
+	(kPublic)
+	(kProtected)
+	(kPrivate)
+
+	(kStrict)
+	(kRequired)
+	(kOptional)
+
+	(kConstref)
+] @type.qualifier
+
+[
+	(kPacked)
+
+	(kAbsolute)
+] @storageclass
+
+(kUses) @include
```

Comment:
On the surface this looks reasonable, although to be honest it would personally bother me slightly if `uses` was highlighted differently from other keywords like `unit`, `interface` and `implementation`. In Pascal, a file has a very specific structure that usually goes something like this:

```pascal
unit bla;
interface
uses
  ...
type
  ...
var/const
  ...
implementation
uses
  ...
end.
```

This is almost like chapters in a book. I wouldn’t want to make one of those keywords stand out more/less than the others. You could argue that this is a styling issue and not a semantic issue – in other words: If I want them to look the same, I could just choose a theme that highlights them the same. However, I might still want to have them highlighted distinctly for other languages. So maybe there is a semantic component to it after all.

Idk, it’s not a deal-breaker or anything, might just be me 🤷

Reply:
You can always link `@include.pascal` to `@keyword` locally so it has the same highlights.

Reply:
That would be possible, but it's an extra step to take. I don't know if it is worth making a change only for users to undo it. Of course, I can't speak for everybody. Maybe others like this change.

Reply:
I think I'd rather keep it consistent with other languages which use `@include`.
If many users don't like it, we can always change it back. ¯\\_(ツ)\_/¯